package.json

```diff
@@ -4,7 +4,7 @@
   "description": "TLS for FxA Pairing",
   "main": "index.js",
   "scripts": {
-    "build": "webpack --env=build",
+    "build": "webpack --env=build && sed -i'' -e '/\\/\\/# sourceMappingURL.*/d' dist/FxAccountsPairingChannel.js",
```

Comment:
@vladikoff I had to remove the space between `-i` and `''` to get this running on my machine.  Might be a mac-vs-wsl thing? Please check whether this new version still works on your machine.

Reply:
yeap! 👍